frontend-next-migration/src/preparedPages/ClanPages/ui/_components/clanoverview/clanViewAndSearch/ClanViewAndSearch.tsx

```diff
@@ -65,43 +65,47 @@ const ClansSearchAndViewDesktop = () => {
-                                        default:
-                                            bgColor = 'rgba(0,0,0,0.6)';
-                                            break;
-                                    }
-
-                                    return (
-                                        <tr
-                                            key={idx}
-                                            style={{ backgroundColor: bgColor }}
-                                            onClick={() => onClickToClan(clan?._id)}
-                                        >
-                                            <td>{clan?.name}</td>
-                                            <td>{clan?.playerCount}</td>
-                                        </tr>
-                                    );
-                                })}
-                            </tbody>
-                        </table>
-                    </div>
-                )}
-            </div>
+            {!error ? (
+                <div className={cls.testContainer}>
+                    {isLoading ? (
+                        <p>{t('loading')}</p>
+                    ) : (
+                        <div className={cls.tableContainer}>
+                            <table className={cls.clanList}>
+                                <thead>
+                                    <tr>
+                                        <th>{t('clan')}</th>
+                                        <th>{t('members')}</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {clans?.data.Clan.map((clan, idx) => {
+                                        let bgColor;
+                                        switch (idx) {
+                                            default:
+                                                bgColor = 'rgba(0,0,0,0.6)';
+                                                break;
+                                        }
+
+                                        return (
+                                            <tr
+                                                key={idx}
+                                                style={{ backgroundColor: bgColor }}
+                                                onClick={() => onClickToClan(clan?._id)}
+                                            >
+                                                <td>{clan?.name}</td>
+                                                <td>{clan?.playerCount}</td>
+                                            </tr>
+                                        );
+                                    })}
+                                </tbody>
+                            </table>
+                        </div>
+                    )}
+                </div>
+            ) : (
+                <h2 style={{ textAlign: 'left', marginBottom: '20px' }}>{t('no_result')}</h2>
```

Comment:
Declare styling inside SCSS, move it there. Why `marginBottom`, its not doing anything here since there are no elements under it?

Reply:
i just looked at how it was done in clanallsubpage.tsx and copied it from there

i also fixed the inline css from there in the latest commit

Reply:
Yeah, the entire file should really be refactored since its quite old and very crutch, but this issue isnt about that.